core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -1220,4 +1220,13 @@ void asyncStoreShutdownTimeoutTooHigh(long configuredAsyncStopTimeout,
    @LogMessage(level = WARN)
    @Message(value = "Distributed task failed at %s. The task is failing over to be executed at %s", id = 330)
    void distributedTaskFailover(Address failedAtAddress, Address failoverTarget, @Cause Exception e);
+
+   @Message(value = "The 'site' must be specified.", id = 331)
+   CacheConfigurationException backupSiteNullName();
+
+   @Message(value = "It is required to specify a 'failurePolicyClass' when using a custom backup failure policy.", id = 332)
+   CacheConfigurationException customBackupFailurePolicyClassNotSpecified();
+
+   @Message(value = "'two-phase-commit' can only be used with SYNC backup strategy.", id = 333)
```

Comment:
The same error message is used both with an XML config and with programmatic configuration, so I'd prefer error messages that do not reference the exact attribute name. E.g. "Using a custom failure policy requires a failure policy class".